### PR TITLE
Reorganize FAQ

### PR DESCRIPTION
**Pull request by ahuang11**
Start FAQ with questions about common things like how to label, re-range, adjust size, legend, and exporting, then some miscellaneous questions, and lastly advanced customization.

### CODE REVIEW

File: doc/FAQ.rst [hunk 1]
```diff
@@ -197,26 +270,44 @@ consistent across HoloViews backends, but in general HoloViews tries
...
+
+**Q: Why are the sizing options so different between the Matplotlib
+and Bokeh backends?"**
+
+**"A:** The way plot sizes are computed is handled in radically
+different ways by these backends, with Matplotlib building plots 'inside
+out' (from plot components with their own sizes) and Bokeh building
+them 'outside in' (fitting plot components into a given overall size).
+Thus there is not currently any way to specify sizes in a way that is
+comparable between the two backends.
```
Comment (jbednar) [inline doc/FAQ.rst]: This bit seems like it should move up to be merged with the earlier question about how to size plots in mpl and bokeh.

**Comment by ahuang11** [comment]
@jbednar only the ordering has changed!

Here's a quick check with Word:
2340 words on left (original); 2340 words on right (reorganized)
image

File: doc/FAQ.rst (hunk 1 above)
Comment (ahuang11) [reply doc/FAQ.rst]: maybe keep this PR for just reorganization and do this in another PR?

Comment (jbednar) [reply doc/FAQ.rst]: Sounds good!

**Comment by jlstevens** [comment]
Makes a lot of sense to me!

**Comment by jbednar** [comment]
I can't see why the tests would be failing for this PR; I've restarted them, but to no avail...

**Comment by ahuang11** [comment]
```
======================================================================
FAIL: test_Bokeh_Elements_display_005 (__main__.NBTester)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ioam/holoviews/doc/nbpublisher/nbtest.py", line 596, in data_comparison
    raise e
AssertionError: Display output mismatch: [{'values_changed': {"root['2211']['roots']['references'][8]['attributes']['data']['y']['__ndarray__']": {'new_value': u'AAAAAAAA8D8/RaOyEf/vP/hTathG/O8/pBrsmp/37z/Hy3Y/HPHvP7vVCye96O8/4T1azoLe7z+cX7fNbdLvP3oPFtl+xO8/FyP8v7a07z9DXnZtFqPvPznGCuiej+8/nVupUVF67z8zPJvnLmPvP0AtcAI5Su8/rI/qFXEv7z8Tv+mw2BLvPwHdUn1x9O4/uAn4Pz3U7j/pC37YPbLuP/doQEF1ju4/Uu8zj+Vo7j+ttMfxkEHuP9iJxLJ5GO4/KOYqNqLt7T9cTA/6DMHtPyQrdZa8ku0/XTsovbNi7T9ZXpQ59TDtP3b+m/CD/ew/dfRs4GLI7D8Z9FMglZHsP6KCjuAdWew/zHkbagAf7D8gGooeQOPrP2mvx3fgpes/NcrrB+Vm6z98EQN5USbrP2av2Iwp5Oo/cly+HHGg6j8/DFMZLFvqPz8/SIpeFOo/zPwljgzM6T8UeQ1aOoLpP3RqejnsNuk/5hEDjibq6D8y+hbP7ZvoP79xvIlGTOg/z8JMYDX75z8uLi8Kv6jnPzysklPoVOc/hXgmHbb/5j/9a9FbLanmPxoqaBhTUeY/KCVibyz45T8cfo2Qvp3lP2jEwb4OQuU/SJqRTyLl5D8JQvuq/obkPwgZGEupJ+Q/+gTLuyfH4z9J2G2af2XjP1yyfZW2AuM/kmBGbNKe4j/3xIzu2DniP5dHOPzP0+E/gVf7hL1s4T+dAPuHpwThP16bdROUm+A/oJtoRIkx4D+3B2uMGo3fPxQEjKRMtd4/DXdeX7Xb3T+7mqhlYQDdP9EMEHpdI9w/lcFbeLZE2z9/gLRUeWTaP7n14xqzgtk/vWOS7XCf2D84AIMFwLrXP9
======================================================================
FAIL: test_Bokeh_Elements_display_006 (__main__.NBTester)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ioam/holoviews/doc/nbpublisher/nbtest.py", line 596, in data_comparison
    raise e
AssertionError: Display output mismatch: [{'dictionary_item_removed': set(["root['3015']['roots']['references'][62]['subtype']", "root['3015']['roots']['references'][62]['attributes']['min_border_top']", "root['3015']['roots']['references'][62]['attributes']['x_scale']", "root['3015']['roots']['references'][62]['attributes']['y_scale']", "root['3015']['roots']['references'][62]['attributes']['plot_height']", "root['3015']['roots']['references'][62]['attributes']['min_border_left']", "root['3015']['roots']['references'][62]['attributes']['plot_width']", "root['3015']['roots']['references'][62]['attributes']['left']", "root['3015']['roots']['references'][62]['attributes']['renderers']", "root['3015']['roots']['references'][62]['attributes']['x_range']", "root['3015']['roots']['references'][62]['attributes']['y_range']", "root['3015']['roots']['references'][62]['attributes']['below']", "root['3015']['roots']['references'][62]['attributes']['toolbar']", "root['3015']['roots']['references'][62]['attributes
```
no idea xD

**Comment by philippjfr** [comment]
Not sure what's up with those test failures, seeing them on other PRs too. I'll update the test data and then rebase this PR once it's fixed.

**Comment by ahuang11** [comment]
Run tests

**Comment by philippjfr** [comment]
@ahuang11 Would you mind rebasing the PR against current master? I've rebuilt the test data so the tests should pass then.

**Pull request merged**